src/patternfly/components/Popover/popover.scss

```diff
@@ -5,13 +5,48 @@
   --pf-c-popover--MaxWidth: calc(var(--pf-c-popover__content--PaddingLeft) + var(--pf-c-popover__content--PaddingRight) + #{pf-size-prem(300px)});
   --pf-c-popover--BoxShadow: var(--pf-global--BoxShadow--md);
 
+  // States
+  --pf-c-popover--m-danger__title-icon--Color: var(--pf-global--danger-color--100);
+  --pf-c-popover--m-warning__title-icon--Color: var(--pf-global--warning-color--100);
+  --pf-c-popover--m-success__title-icon--Color: var(--pf-global--success-color--100);
+  --pf-c-popover--m-info__title-icon--Color: var(--pf-global--info-color--100);
+  --pf-c-popover--m-default__title-icon--Color: var(--pf-global--default-color--200);
+
+  @media (min-width: $pf-global--breakpoint--xl) {
+    --pf-c-popover--m-align-top--spacer: var(--pf-c-popover--m-align-top--xl--spacer);
+  }
+
   // Content
   --pf-c-popover__content--BackgroundColor: var(--pf-global--BackgroundColor--100);
   --pf-c-popover__content--PaddingTop: var(--pf-global--spacer--md);
   --pf-c-popover__content--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-popover__content--PaddingBottom: var(--pf-global--spacer--md);
   --pf-c-popover__content--PaddingLeft: var(--pf-global--spacer--md);
 
+  // Title
+  --pf-c-popover__title--LineHeight: var(--pf-global--LineHeight--sm);
+  --pf-c-popover__title--FontFamily: var(--pf-global--FontFamily--heading--sans-serif);
+  --pf-c-popover__title--FontSize: var(--pf-global--FontSize--md);
+
+  // Title Default
+  --pf-c-title--m-default__title--Color: var(--pf-global--default-color--200);
```

Comment:
I don't think you'll need these variations on the title, though fwiw we only keep the parent component vars here, so we would have these in the title component instead of the popover component.

Reply:
Thank you @mcoker, will this also be addressed in #4123? if I remove the variations then the icons for popover variations will lose the colours as below:

![after](https://user-images.githubusercontent.com/4890675/125684181-54b66968-9436-445a-8964-c72e881b7ba3.png)